oomph-vscode/README.md

```diff
@@ -0,0 +1,7 @@
+# oomph-vscode
+
+syntax highlighting for oomph in VS code
+
+# Install
+
+clone this directory to `~/.vscode/extensions/`
```

Comment:
Can you combine this into oomph's main `README.md`, near the porcupine stuff?

Reply:
This file can be now deleted, since everything it says is also in the main README.md.